core/audits/origin-isolation.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {Audit} from './audit.js';
+import {MainResource} from '../computed/main-resource.js';
+import * as i18n from '../lib/i18n/i18n.js';
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that evaluates the security of a page's COOP header for origin isolation. "COOP" stands for "Cross-Origin-Opener-Policy". */
+  title: 'Ensure the proper usage of the COOP header to isolate the origin.',
+  /** Description of a Lighthouse audit that evaluates the security of a page's COOP header for origin isolation. This is displayed after a user expands the section to see more. No character length limits. The last sentence starting with 'Learn' becomes link text to additional documentation. "COOP" stands for "Cross-Origin-Opener-Policy". */
+  description: 'Deployment of the COOP header allows isolation of the top-level document to not share a browsing context group with cross-origin documents. ' +
```

Comment:
Both title and description include "COOP header", but neither say "Cross-Origin-Opener-Policy". I'd never heard of this header prior to reading this PR, sooo… maybe a good idea to revise the description.

Reply:
Ah, yeah, good point. I will fix it in the code as well, if @adamraine doesn't think it's too long and we should re-phrase it generally.

Reply:
In the long form description, we should use "Cross-Origin-Opener-Policy (COOP)" to identify the acronym. Everywhere else it should be fine to use "COOP"

Reply:
SGTM, thanks Adam!